sites/friday/src/get-started/here/configure/nodejs.md

```diff
@@ -0,0 +1,169 @@
+---
+title: JavaScript/Node.js
+weight: 10
+# description: All you need to know about creating a new project with {{% vendor/name %}}
+---
+
+When dealing with Javascript/Node.js stacks, the information below may help customize your configuration.
+These sections provide Javascript/Node.js-specific configuration details, but also be sure to consult the common Upsun documentation as well:
+
+- [Configuring applications](/create-apps)
+- [Setting up managed services](/add-services)
+- [Handling requests](/define-routes)
+
+## Build flavors
+
+By default, Upsun will make assumptions about how you want to build your application. 
+Namely, that you are managing your dependencies with npm, and that the very first thing you'd like to run is a particular and common production flavor of `npm install`.
+
+This is called a build `flavor`, but it's assumption may prove inappropriate for your application and cause your builds to fail (such as if you'd like to use yarn or bun instead of npm).
+[Consult the documentation](/languages/nodejs#dependencies) to learn more about disabling this feature.
+
+## Available package managers
+
+Certain package managers come pre-installed on all Upsun `nodejs` container types:
+
+* *npm*; example: ``npm install platformsh-config``
+* *npx*; example: ``npx create-strapi-app <APP_NAME>``
+* *bun*; example: ``bun install platformsh-config``
+
+Yarn is an exception, as it [requires that you install it explicitly](/languages/nodejs#use-yarn-as-a-package-manager). 
```

Comment:
Yarn **is** bundled with node via corepack but isn't _accessible_ expect through the use of corepack. Users dont have to install it but they do have to explicitly declare its use in the package.json file.